### ISSUE TITLE
titleTemplate with arrow function expression produces an error

### ISSUE THREAD

**Issue by pbastowski** [nuxt/nuxt.js#5051]
### Version

[v2.4.3](https://github.com/nuxt.js/releases/tag/v2.4.3)

Browser: Chrome 72.0.3626.109
OS: MacOS 10.13.6 High Sierra

### Reproduction link

[https://codesandbox.io/s/rmmo942z1p](https://codesandbox.io/s/rmmo942z1p)

### Steps to reproduce

The following `titleTemplate` arrow function produces an error, see below.

### This does not work
```js
titleTemplate: title => (title ? `${title} - ${pkg.name}` : pkg.name),
```

### This works
The only difference is the explicit `return` statement.
```js
titleTemplate: title => {
    return title ? `${title} - ${pkg.name}` : pkg.name
},
```

### Error
![image](https://user-images.githubusercontent.com/5649024/52912740-c3fb3380-32b5-11e9-9138-b741627863ca.png)

### What is expected ?

titleTemplate function works

### What is actually happening?

titleTemplate function produces a compile error

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c8676">#c8676</a>)</em></sub></div>

**Comment by R4wizard** [comment]
To expand upon this, the following three things do *not* work for me either:

```
const TitleTemplate = title => 'MY SITE NAME' + (title ? ' :: ' + title : '');

module.exports = {
  head: {
    titleTemplate: TitleTemplate
  }
};
```

or

```
const TitleTemplate = title => {
	return ('MY SITE NAME' + (title ? ' :: ' + title : ''));
};

module.exports = {
  head: {
    titleTemplate: TitleTemplate
  }
};
```

or

```
module.exports = {
  head: {
    titleTemplate: title => {
      return (DefaultPageTitle + (title ? ' :: ' + title : ''));
    }
  }
}
```

**Comment by pbastowski** [comment]
Yeah, it looks like the transpiler does not know how to properly transpile an arrow function that directly returns an expression.

**Comment by manniL** [comment]
Maybe related to #4283, #4558 and so on

**Comment by pbastowski** [comment]
This is still not working properly. I just checked the provided codesandbox, above, with Nuxt version 2.5.1 and got the following error:

![image](https://user-images.githubusercontent.com/5649024/54930922-6b543180-4f18-11e9-8e82-08935d3fbc39.png)


Why is this issue closed?

**Comment by manniL** [comment]
cc @galvez

**Comment by galvez** [comment]
@pbastowski thanks for reporting this, I'll look into it.

**Comment by pi0** [comment]
Fixed by @galvez  with #5580. Will be fixed in v2.6.3.
